### ISSUE TITLE
Output file job cleanup time is always $null

### ISSUE THREAD

**Issue by ashdar** [sqlcollaborative/dbachecks#633]
### General Troubleshooting steps
- [X] Verified running the latest release of dbachecks? 

Does `(Find-Module dbachecks).Version match (Get-Module dbachecks).Version.ToString()`

![image](https://user-images.githubusercontent.com/6729780/37113488-60e5e172-21fa-11e8-8115-814edfbfabb6.png)

- [ ] Verified errors are not related to permissions?
- [ ] Can duplicate in new/clean PowerShell session (clean = `powershell -NoProfile`)?

### Version Information
 - Operating System (Name|Version): Windows 7
 - PowerShell Version: 5.1
 - SQL Server (Edition|Version): Standard & Enterprise, SQL 2008 and SQL 2008R2

### Steps to Reproduce
I just upgraded this morning from 1.1.173 to 1.2.5. 
Run the checks for the Maintenance solution, including the OutputFileCleanup checks.
DbaChecks seems to think that the '$days' value in the command line that the job runs is $null. When compared the the 30 days stored in config policy.ola.outputcleanup, the test fails.

### Description of Bug
I have already dug into Maintenance.Tests.ps1. At first I thought that the configuration value was not being read in from psframework correctly. It turns out that was OK. BUT, the code that greps the command line string from the job step seems to never find the value. The regex works, it finds matches. 

BUT the .Value seems to be syntactically wrong. The Matches hashtable doesn't work that way.

In short, I think that the line: $days = $Matches[1].Value
should be: $days - $Matches[1]

**Comment by ashdar** [comment]
I occurred to me that submitting a patch would be easier than someone trying to decode what I wrote last week.

**Comment by SQLDBAWithABeard** [comment]
closed with #634
